onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -133,6 +133,9 @@
         <activity android:name="org.onebusaway.android.ui.RegionsActivity"/>
         <activity android:name="org.onebusaway.android.ui.AgenciesActivity"/>
 
+        <activity android:name=".report.ui.ReportActivity"/>
```

Comment:
Let's list the fully qualified activity name here to match the other activities in the manifest.